lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -399,8 +399,7 @@ sub _schedule_iso {
             $_ => [
                 [keys %{$tmp_downloads{$_}->{destination}}], $tmp_downloads{$_}->{do_extract},
                 $tmp_downloads{$_}->{blocked_job_id}]
-          }
-          keys %tmp_downloads;
+        } keys %tmp_downloads;
```

Comment:
It wouldn't hurt to create a separate commit just for style changes. (This also counts for style changes in `create_downloads_list`.)